back-end/api/api/settings.py

```diff
@@ -26,7 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["https://cyver.herokuapp.com", 'http://127.0.0.1']
```

Comment:
Can we make this line like this?
```suggestion
ALLOWED_HOSTS = ["*"]
```
NOTE: I haven't tested this.

https://docs.djangoproject.com/en/3.1/ref/settings/#allowed-hosts

Currently I'm not sure if other's deployment will work if we only allow for "https://cyver.herokuapp.com" and localhost, since we'll have different domains.

Making it "*" _should_ allow anyone to host it regardless of what our Heroku hostname is. It's not secure, but for the sake of the project this should be fine.

Could you see if "*" works?

Reply:
Yes I'll do it now

Reply:
It works